modules/luabackend/reload.cc

```diff
@@ -143,6 +143,8 @@ void LUABackend::reload() {
 		get_lua_function(lua, "removedomainkey", &f_lua_removedomainkey);
 		get_lua_function(lua, "activatedomainkey", &f_lua_activatedomainkey);
 		get_lua_function(lua, "deactivatedomainkey", &f_lua_deactivatedomainkey);
+        get_lua_function(lua, "hidedomainkey", &f_lua_hidedomainkey);
+        get_lua_function(lua, "unhidedomainkey", &f_lua_unhidedomainkey);
```

Comment:
space vs tabs issue?

Reply:
Since the luabackend is gone this is fixed.